nominal/nominal.py

```diff
@@ -596,19 +596,29 @@ def upload_mcap_video(
 
 
 def create_streaming_connection(
-    datasource_id: str,
     connection_name: str,
+    datasource_id: str | None = None,
```

Comment:
moving a positional arg is a breaking change unfortunately, so we'll need to do something different here